src/pzt/admission-controller.yaml

```diff
@@ -0,0 +1,6 @@
+slug: kubernetes-admission-controller
+name: Kubernetes Admission Controller
+template: ingress
```

Comment:
if this is going to share a template with kubernetes ingress, let's rename the template to k8s, or something?

Reply:
@alanchrt thoughts?

Reply:
Yup, agreed. I think it should be `template: kubernetes`. I can try to put a commit on here to do the rename and the wiring.